lib/config/presets/internal/packages.ts

```diff
@@ -51,13 +51,19 @@ export const presets: Record<string, Preset> = {
     matchPackageNames: ['postcss'],
     matchPackagePrefixes: ['postcss-'],
   },
+  testingLibrary: {
```

Comment:
Why a new preset? Simply add `matchPackagePrefixes` to the other preset.

Reply:
I thought it would be useful to have them as a standalone package as well, but yeah, I can change that no problem